lib/transforms/subsetFonts.js

```diff
@@ -599,8 +609,43 @@ module.exports = ({
       }
     }
 
-    // Generate subsets:
+    // Generate codepoint sets for original font, the used subset and the unused subset
+    for (const htmlAssetTextWithProps of htmlAssetTextsWithProps) {
+      for (const fontUsage of htmlAssetTextWithProps.fontUsages) {
+        const usedCodepoints = fontUsage.text
+          .split('')
+          .map(c => c.codePointAt(c[0]));
+
+        fontUsage.codepoints = {
+          used: usedCodepoints
+        };
+
+        const originalFont = assetGraph.findAssets({
+          url: fontUsage.fontUrl
+        })[0];
+
+        let originalCodepoints;
+        try {
+          originalCodepoints = fontkit.create(originalFont.rawSrc).characterSet;
+        } catch (error) {
+          // FIXME: Should this emit a warning or just swallow?
+          // A warning is already emitted in the subsetting code
```

Comment:
On my PR I ended up swallowing it: https://github.com/assetgraph/assetgraph/pull/899/files#diff-72ec0ab8735cbd5d6ed5917c7a1a44a5R639

Otherwise the test with the unknown/mangled font fails.

I think it's fine if the warning has already been emitted.